subcommittees/mentoring/files/discussion_recruiting_email.txt

```diff
@@ -0,0 +1,13 @@
+Greetings, all!  
+
+This is an invitation to contribute to the Software and Data Carpentry communities in a very low-commitment, lots-of-fun way!  
```

Comment:
I mean low-commitment as in, 1 hour every few weeks.  I'll see if I can reword to make that clearer.